packages/core/src/identity/Identity.ts

```diff
@@ -123,7 +123,7 @@ export default class Identity {
    * ```
    */
   public static buildFromSeed(seed: Uint8Array): Identity {
-    const keyring = new Keyring({ type: 'ed25519' })
+    const keyring = new Keyring({ type: 'sr25519' })
```

Comment:
That is a bigger change, we should maybe do separately, or is this needed for parachains?

Reply:
This is only needed to have balances on the parachain since the endowed accounts are sr25519 accounts. Would do that in this PR either, but this was needed to test with the parachain.